### PR TITLE
Update Define Spacing

### PR DESCRIPTION
**Pull request by ZephyrTFA**
UPDATES A LOT OF SPACING FOR DEFINES TO BE IN LINE WITH CODING GUIDELINES.
THIS WAS DONE VIA REGEX AND WILL REQUIRE AT LEAST A WEEK OF BEING TM'D BEFORE FULL MERGING

### CODE REVIEW

File: code/__DEFINES/flags.dm [hunk 1]
```diff
@@ -45,90 +45,90 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
...
+#define CULT_PERMITTED_1 (1<<3)
 /// Blocks lava rivers being generated on the turf
-#define NO_LAVA_GEN_1				(1<<6)
+#define NO_LAVA_GEN_1 (1<<6)
 /// Blocks ruins spawning on the turf
-#define NO_RUINS_1					(1<<10)
+#define NO_RUINS_1 (1<<10)
 
-////////////////Area flags\\\\\\\\\\\\\\
+////////////////Area flags\\\\\\\\\\\\\ \
```
Comment (martinlyra) [inline code/__DEFINES/flags.dm]:
```suggestion
////////////////Area flags\\\\\\\\\\\\\\
```

The CI is complaining about this

**Review by martinlyra — CHANGES_REQUESTED** [review]

File: code/__DEFINES/flags.dm (hunk 1 above)
Comment (ZephyrTFA) [reply code/__DEFINES/flags.dm]: My savior

**Review by martinlyra — APPROVED** [review]

File: code/game/machinery/scan_gate.dm [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-#define SCANGATE_FLY			"fly"
-#define SCANGATE_PLASMAMAN		"plasma"
-#define SCANGATE_MOTH			"moth"
-#define SCANGATE_JELLY			"jelly"
-#define SCANGATE_POD			"pod"
-#define SCANGATE_GOLEM			"golem"
-#define SCANGATE_ZOMBIE			"zombie"
-#define SCANGATE_SPIDER			"rachnid"
+#define SCANGATE_HUMAN 		"human"
+#define SCANGATE_lizard "lizard"
```
Comment (martinlyra) [inline code/game/machinery/scan_gate.dm]:
```suggestion
#define SCANGATE_LIZARD "lizard"
```

**Review by martinlyra — CHANGES_REQUESTED** [review]

**Comment by martinlyra** [comment]
Just discard your changes on the `check_regex.py`, reflect your new target(s) to the `check_regex.yaml` file. That should be good to go!

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: bf2a3d7daffd8042d3adbe075ec803cd3abd4e03
Pull Request: 289cc24feb05cc64ea8e9b2427e42845e60eed8c
Server: ca0ff8b9f76846976a9d8d2ec5e18cf60ecd28be

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by MarkSuckerberg** [comment]
LGTM

**Comment by martinlyra** [comment]
:tada:

**Pull request merged**